### PR TITLE
Trial Update dco.yml

### PR DESCRIPTION
**Pull request by mojito317**
## PR summary

Testing the CODEOWNERS feature initialized in #29. Won't be merged.

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [ ] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [x] Other (please describe)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by bessbd — APPROVED** [review]

**Review by emlaver — APPROVED** [review]

**Comment by mojito317** [comment]
After the approval of @bessbd and @emlaver I can say the #29 approach works well, it is enough to get at least 1 of the group members approval:
image

After the test, I'm closing this PR.

**Pull request closed without merging**
